webrender/src/frame.rs

```diff
@@ -301,65 +301,86 @@ impl Frame {
             None => return false,
         };
 
-        let layer = self.layers.get_mut(&scroll_layer_id).unwrap();
-        if layer.scrolling.started_bouncing_back && phase == ScrollEventPhase::Move(false) {
-            return false
-        }
+        let scroll_root_id = match scroll_layer_id.info {
+            ScrollLayerInfo::Scrollable(_, scroll_root_id) => scroll_root_id,
+            ScrollLayerInfo::Fixed => unreachable!("Tried to scroll a fixed position layer."),
```

Comment:
I think in this case, get_scroll_layer is guaranteed to never return a position: fixed layer, so the unreachable!() is probably correct here.